kolibri/core/content/management/commands/exportcontent.py

```diff
@@ -91,19 +94,23 @@ def handle_async(self, *args, **options):
 
         self.update_job_metadata(total_bytes_to_transfer, total_resource_count)
 
-        exported_files = []
-
-        with self.start_progress(
-            total=total_bytes_to_transfer
-        ) as overall_progress_update:
-            for f in files:
-
-                if self.is_cancelled():
-                    break
-
-                dest = self.export_file(f, data_dir, overall_progress_update)
-                if dest:
-                    exported_files.append(dest)
+        # dont copy files if we are only exporting the manifest
+        if not options["manifest_only"]:
+            logger.info(
+                "Exporting content for channel id {} to {}".format(channel_id, data_dir)
+            )
+            exported_files = []
```

Comment:
How about moving this copying part to a separate method?

Reply:
Yes, that seems fine to me.

Reply:
Alright will do that!